core/spec/lib/spree/money_spec.rb

```diff
@@ -268,4 +268,32 @@
       end
     end
   end
+
+  describe "<=>" do
```

Comment:
Yes to testing that `Comparable` is included 👍 

No need to test a well tested module from Rubys standard lib